### PR TITLE
ODC-7613: Update the UI for selecting the Git Provider in the Import form

### PR DESCRIPTION
**Pull request by Lucifergene**
**Fixes**: 
https://issues.redhat.com/browse/ODC-7613

**Solution Description**: 
- [x] Update the Dropdown component to Tiles
- [x] Make sure the validation happens once a new Git type is selected
- [x] Adjust CSS of the Tile to adapt to the order of the Import form
- [x] Add Gitea Icon React component

**Screen shots / Gifs for design review**: 

Screenshot from 2024-06-06 02-33-10
Screenshot from 2024-06-06 02-34-02

https://github.com/openshift/console/assets/47265560/10d83848-2306-40e8-83fa-dddc5c25c576

**Unit test coverage report**: 

- [x] E2E Tests added

**Test setup:**
1. Import from Git Form -> "https://open-bitbucket.nrao.edu/projects/PIPE/repos/pipeline/browse"
2. Click on the "Bitbucket" git type
3. Select the Python Builder Image
4. Submit the form

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/dev-console/src/components/import/git/GitSection.tsx [hunk 1]
```diff
@@ -646,15 +641,16 @@ const GitSection: React.FC<GitSectionProps> = ({
...
+          {/* <DropdownField
             name={`${fieldPrefix}git.type`}
             label={t('devconsole~Git type')}
             items={GitReadableTypes}
             title={GitReadableTypes[values.git.type]}
             fullWidth
             required
             dataTest="git-type"
-          />
+          /> */}
```
Comment (vikram-raj) [inline frontend/packages/dev-console/src/components/import/git/GitSection.tsx:653]: nit: remove this commented code.

Comment (Lucifergene) [reply frontend/packages/dev-console/src/components/import/git/GitSection.tsx]: Thanks. Removed it.

**Review by vikram-raj — COMMENTED** [review]
/lgtm

**Comment by vikram-raj** [comment]
propagate from the epic

/label docs-approved
/label px-approved

File: frontend/packages/dev-console/src/components/import/GiteaIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+const GiteaIcon: React.FC<React.HTMLProps<SVGElement>> = ({ style }): React.ReactElement => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 20 650 450"
+      fill="currentColor"
+      aria-labelledby="icon-title-110"
```
Comment (logonoff) [inline frontend/packages/dev-console/src/components/import/GiteaIcon.tsx:9]:
```suggestion
      aira-hidden="true"
```

`icon-title-110` no longer exists

**Review by logonoff — APPROVED** [review]

**Comment by sanketpathak** [comment]
@Lucifergene The text under the git URL says dropdown menu which has to be updated
Screenshot 2024-06-21 at 4 07 22 AM

**Comment by sanketpathak** [comment]
Verified on a cluster with admin and non-admin users
Screenshot 2024-06-22 at 5 53 53 PM
/label qe-approved

**Pull request merged**